examples/spouse_example/postgres/incremental/spouse_example.f1.active.rules

```diff
@@ -1,3 +1,3 @@
 # List names of the inference rule to mark as active in materialization phase here.
 # It's in the form of the relation name of the head in the inference rule followed by an underscore and the zero-based index of the rule.
-has_spouse_0  # You can put comments after a hash letter.
+inference_rule_0  # You can put comments after a hash letter.
```

Comment:
This is another place where the compiler naming convention needs to be fixed as pointed out in #411.